src/jira/util/jwt.ts

```diff
@@ -81,7 +81,16 @@ function decodeAsymmetricToken(token: string, publicKey: string, noVerify: boole
 
 function verifyQsh(qsh: string, req: Request): boolean {
```

Comment:
Can we have some test for this bug?

Reply:
Test case for this case has been added in `jwt.test.ts`